vmm/src/pci_segment.rs

```diff
@@ -347,11 +347,6 @@ impl Aml for PciSegment {
                     &aml::AddressSpace::new_bus_number(0x0u16, 0x0u16),
                     #[cfg(target_arch = "x86_64")]
                     &aml::Io::new(0xcf8, 0xcf8, 1, 0x8),
-                    &aml::Memory32Fixed::new(
```

Comment:
@michael2012z You were the one who added this originally (and I made it segment aware later) in 5f27d649a686b284955e39b45376821586bd979e but there was no explanation why. Testing seems to indicate that it's fine to remove.

Reply:
Cannot recall why I had to add this.
But now that test is good, just continue removing it.